Assets/VRTK/Scripts/Controls/2D/PanelMenu/PanelMenuController.cs

```diff
@@ -0,0 +1,517 @@
+// Panel Menu Controller|Prefabs|0070
+namespace VRTK
+{
+    using System.Collections;
+    using UnityEngine;
+
+    /// <summary>
+    /// Purpose: top-level controller class to handle the display of up to four child PanelMenuItemController items which are displayed as a canvas UI panel.
+    /// </summary>
+    /// <remarks>
+    ///    > This script should be attached to an VRTK_InteractableObject > first child GameObject [PanelMenuController].
+    ///    > The [PanelMenuController] must have a child GameObject [panel items container].
```

Comment:
also, do you want this all as an indented note field?

I'd remove the ` >` because it will just become a paragraph in the instructions, which will read better